java/test/src/main/java/io/ray/test/RuntimeEnvTest.java

```diff
@@ -81,7 +79,6 @@ public void testPerActorEnvVars() {
   }
 
   public void testPerActorEnvVarsOverwritePerJobEnvVars() {
```

Comment:
Maybe we need to simplify or delete this test case.

Reply:
deleted.